inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaClassElement.java

```diff
@@ -431,7 +435,7 @@ public <T extends io.micronaut.inject.ast.Element> List<T> getEnclosedElements(@
      * {@link io.micronaut.inject.ast.ElementFactory#newSourceMethodElement(ClassElement, Object, ElementAnnotationMetadataFactory)}.
      *
      * @param query The query
-     * @param <T>   The element type
```

Comment:
you have 2 styles in these classes. Do we need `Align parameter / thrown exception decriptions` or not?

Reply:
@graemerocher could you tell me which codestyle is right?